docs/source/telegram.dice.rst

```diff
@@ -3,4 +3,4 @@ telegram.Dice
 
 .. autoclass:: telegram.Dice
     :members:
-    :show-inheritance:
\ No newline at end of file
+    :show-inheritance:
```

Comment:
yes. I added sorting by source, then removed it but left a blank line at EOF. I checked out `master`, the same file there has no blank line at EOF. But since most (all?) other files do, then we might as well let this one have one too :)